api/envoy/config/route/v3/route_components.proto

```diff
@@ -1110,6 +1110,11 @@ message RouteAction {
   // :ref:`HTTP_DOWNSTREAM_STREAM_IDLE <envoy_v3_api_enum_value_config.overload.v3.ScaleTimersOverloadActionConfig.TimerType.HTTP_DOWNSTREAM_STREAM_IDLE>`.
   google.protobuf.Duration idle_timeout = 24;
 
+  // Specifies how to send request over TLS early data.
+  // If absent, allows `safe HTTP requests <https://www.rfc-editor.org/rfc/rfc7231#section-4.2.1>`_ to be sent on early data.
+  // [#extension-category: envoy.route.early_data_option]
+  core.v3.TypedExtensionConfig early_data_option = 39;
```

Comment:
Your call, but I think "policy" might be a better choice than "option" (here and elsewhere)

Reply:
renamed